src/handlers/lib/kafka/testProducer.js

```diff
@@ -0,0 +1,106 @@
+// @TODO to be cleaned up, used for testing the handlers
```

Comment:
Once again, this should be moved to the test folder 💃 

Perhaps under 'test/util' folder?

Reply:
@rajivmothilal still same dir?